### DISCUSSION TITLE
galaxy brain

### DISCUSSION THREAD

**Discussion by s0meee** [Schweinepriester/achievement-testarea#33] [Q&A] [answered]
.

**Comment by 1wos** [comment, marked as answer]
test

**Reply by hoseinpur** [reply]
Please mark this to see if it's working!

**Reply by 1wos** [reply]
I checked your reply now. 
If you don't mind, why don't you upload the Galaxy brain posts together and assign answers?

**Reply by 1wos** [reply]
I'll post a new post now, so please reply to it!  I'll take it as an answer!

**Comment by hoseinpur** [comment]
let's test to see if this works!!

**Comment by 1wos** [comment]
> Please mark this to see if it's working!
https://github.com/Schweinepriester/achievement-testarea/discussions/36

**Reply by seo-asif** [reply]
you already got galaxy brain

**Comment by pulkitxm** [comment]
best Answer
Please mark this

**Reply by 1wos** [reply]
Can you post my repository discord questions for the galaxy brain test? Can you post a question on the Q&A? Then I'll help you through adoption!

https://github.com/sommizzu/Big-Data-Convergence/discussions
